server/controllers/userController.js

```diff
@@ -57,54 +77,61 @@ class UserController {
           response.status(404).send({
             success: false,
             message: 'User not found'
-          })
+          });
         }
       })
-      .catch(error => response.status(401).send(error));
+      .catch(error => response.status(404).send(error));
```

Comment:
Why do you have to return 404 on two occasions? I think something isn't all that right here.